ChaosMod/Effects/db/Player/PlayerBigBang.cpp

```diff
@@ -0,0 +1,92 @@
+#include <stdafx.h>
+
+//Effect by ProfessorBiddle, Based on code from Pongo and Last0xygen
+
+static void OnStart()
+{
+	std::vector<Entity> entities;
+	int maxEntities = 20;
+
+	for (auto ped : GetAllPeds())
+	{
+		if (maxEntities == 10)
+		{
+			break;
+		}
+
+		maxEntities--;
+
+		if (!IS_PED_A_PLAYER(ped) && !IS_ENTITY_A_MISSION_ENTITY(ped))
```

Comment:
this could result in adding less than 10 entities, if the first 10 peds are either an mission entity or player

Reply:
fixed